integration-tests/src/bin/gen_blockchain_data.rs

```diff
@@ -90,11 +90,24 @@ async fn main() {
             .clone()
             .evm_version(EvmVersion::London);
 
-        let compiled = solc
-            .compile(&input)
-            .unwrap_or_else(|_| panic!("solc compile error {:?}", path_sol));
-        if !compiled.errors.is_empty() {
-            panic!("Errors compiling {:?}:\n{:#?}", &path_sol, compiled.errors)
+        // .compile() call will either fail with Err variant or return Ok(CompilerOutput)
+        // which may contain Errors or Warnings
+        let compiled = match solc.compile(&input) {
```

Comment:
I noticed that we only need to add `disable_recursion_limit` to the Deserializer. Using the public method `compile_output` from Solc can achieve that directly.

I think we don't need the Solcwc wrapper because much of the content in the `Solcwc` is about solidity version management, which we don't need here.

I have a local build with the following change that can successfully do `cargo run --bin gen_blockchain_data` on those contracts hitting the recursion limit.

```suggestion
        let output = solc.compile_output(&input).unwrap();
        let mut deserializer: serde_json::Deserializer<serde_json::de::SliceRead<'_>> =
            serde_json::Deserializer::from_slice(&output);
        deserializer.disable_recursion_limit();
        let compiled = match CompilerOutput::deserialize(&mut deserializer) {
```

Reply:
@ChihChengLiang nice! This means that we need neither wcerrors.rs nor worst_case.rs, correct? Since we directly apply the external pub fn compile_output().